app/apollo/resolvers/cluster.js

```diff
@@ -201,16 +212,18 @@ const clusterResolvers = {
 
         await applyQueryFieldsToClusters([cluster], queryFields, args, context);
 
+        logger.info({req_id, user, org_id, clusterName, cluster}, `${queryName} applying query fields`);
```

Comment:
... in this log example, it doesn't seem to add any value (and incorrectly states that it's "applying query fields" when it's really already completed doing so).

Reply:
fixed